Lib/test/test_capi/test_abstract.py

```diff
@@ -1007,6 +1007,46 @@ def test_object_generichash(self):
         for obj in object(), 1, 'string', []:
             self.assertEqual(generichash(obj), object.__hash__(obj))
 
+    def run_iter_api_test(self, next_func):
+        for data in (), [], (1, 2, 3), [1 , 2, 3], "123":
```

Comment:
```suggestion
        for data in [(), [], (1, 2, 3), [1 , 2, 3], "123"]:
```

I know that we can just forget the brackets etc, but I find it easier to read. But maybe it's ebcause I only found that it works only recently (I see that the rest of the file is actually omitting those brackets so I don't know whether you want to keep a consistent style or if you want to make it readable).

Reply:
Both styles are used in the stdlib. I prefer the implicit tuple for tests like this.